python/cudaq.py

```diff
@@ -35,10 +35,6 @@ def quake_synth():
     command = os.path.join(os.path.dirname(__file__), "bin/quake-synth")
     subprocess.call([command] + sys.argv[1:])
 
-def qtx_translate(): 
```

Comment:
You can remove the `nvqpp()`, `cudaq_quake()`, `cudaq_opt()`, `quake_translate()`, `quake_synth()` functions as well.

Reply:
I think Anthony was already working on removing those stale bits.

Reply:
#104